lms/static/js/student_account/components/StudentAccountDeletion.jsx

```diff
@@ -113,11 +107,9 @@ export class StudentAccountDeletion extends React.Component {
                     <span>{bodyDeletion} </span>
                     <span>{bodyDeletion2}</span>
                 </p>
-                <p
-                    className="account-settings-header-subtitle"
-                    // eslint-disable-next-line react/no-danger
-                    dangerouslySetInnerHTML={{ __html: loseAccessText }}
-                />
+                <p className="account-settings-header-subtitle">
+                    <span>{loseAccessText}</span>
+                </p>
```

Comment:
Thanks for updating this! It's much better than using `dangerouslySetInnerHTML`!

I am curious as to why you're putting a `<span>` inside the `<p>` though. Is there something about

```suggestion
                <p className="account-settings-header-subtitle">{loseAccessText}</p>
```

that causes problems?

Reply:
Mmmm it might be because I copied and pasted it. I updated it! Thanks